### DISCUSSION TITLE
Window title problem

### DISCUSSION THREAD

**Discussion by Yucie** [smplayer-dev/smplayer#1023] [Q&A]
![problem](https://github.com/user-attachments/assets/7f053a96-7ba9-4bec-a4b3-a5d4a5882cf3)

I add a m3u8 stream in playlist and name it to something.
When i play the item in list. The window title will show clip info rather than the name in playlist.
Is it possible to show the playlist name rather than loaded stream clip name.

**Comment by smplayer-dev** [comment]
Enable the option "display the name of the media in the window title" in preferences -> advanced.

**Comment by Yucie** [comment]
![problem2](https://github.com/user-attachments/assets/933d5b2e-27f2-4e5e-9983-d37d5c519411)
I set ithe option to enable or disable, the title still display clip info name.

Here is my playlist file.
[example.zip](https://github.com/user-attachments/files/16493915/example.zip)

**Comment by smplayer-dev** [comment]
In the case of a playlist I think it's not possible.

**Reply by Yucie** [reply]
In case of youtube video link. When video loaded, the window title will be change with youtube video title.
Is it possible to match the link with playlist file and then get name to change window title?
